core/src/main/java/org/sonarsource/sonarlint/core/container/global/ExtensionInstaller.java

```diff
@@ -49,8 +50,9 @@
   /**
    * Standalone mode
    */
-  public ExtensionInstaller(SonarRuntime sonarRuntime, PluginRepository pluginRepository, Configuration bootConfiguration, PluginVersionChecker pluginVersionChecker) {
-    this(sonarRuntime, pluginRepository, bootConfiguration, pluginVersionChecker, Collections.emptySet());
+  public ExtensionInstaller(SonarRuntime sonarRuntime, PluginRepository pluginRepository, Configuration bootConfiguration, PluginVersionChecker pluginVersionChecker,
```

Comment:
Instead of 2 different constructors, I think now we can inject `AbstractGlobalConfiguration`